demisto_sdk/dev_tools/runner.py

```diff
@@ -0,0 +1,136 @@
+import re
+
+from demisto_sdk.common.configuration import Configuration
+from demisto_sdk.common.tools import print_error, print_color, LOG_COLORS, print_v, print_warning
+from demisto_sdk.common.rest_api import Client, DEMISTO_API_KEY_ENV
+
+
+SECTIONS_REGEX = re.compile(r'^(Context Outputs|Human Readable section|Raw Response section)')
+FULL_LOG_REGEX = re.compile(r'.*Full Integration Log')
+
+
+class Runner:
+    def __init__(self, query: str, url: str, insecure: bool = False, debug_mode: bool = False,
+                 verbose: bool = False, configuration: Configuration = Configuration()):
+        self.query = query
+        self.base_url = url
+        self.verify_cert = not insecure
+        self.log_verbose = verbose
+        self.debug_mode = debug_mode
+
+    def run(self):
+        """Do the job. Run the integration command on the remote Demisto instance
+        and pretty prints the result.
```

Comment:
please update to: `Run the integration command on the remote Demisto instance and pretty prints the result`

Reply:
Done